vanilla.html

```diff
@@ -65,7 +66,24 @@ <h2 class="">Jobs in Computer Science</h2>
             <p>Jobs that you can get with a Computer Science Degree</p>
           </div>
           <!-- TODO: Add Jobs with Salary -->
-          <p></p>
+          <table class="jobTable">
+            <tr>
+              <th>Job Title</th>
+              <th>Salary(per year)</th>
+            </tr>
+            <tr>
+              <td>Software Developer</td>
```

Comment:
It might be useful to add a hyperlink to these so people could go linked to a site with more info about the role

Reply:
Should I just put the website that I used to get those? https://www.rasmussen.edu/degrees/technology/blog/entry-level-computer-science-jobs/

And then create a link that says sources?

Reply:
If you have a more detailed link for each of them add one for each, if you don't I would add a line under the table that's something like "Data gathered from this source" with 'source' being a hyperlink (same text style, just add <a> tag around it so it links and underlines).

Reply:
Changes made.